BioArchLinux/spades/PKGBUILD

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Maintainer: PumpkinCheshire <me at pumpkincheshire dot top>
+# Maintainer: alienzj <alienchuj at gmail dot com>
+
+pkgname=spades
+pkgver=3.15.5
+pkgrel=1
+pkgdesc='SPAdes Genome Assembler'
+url='http://cab.spbu.ru/software/spades/'
+arch=('x86_64')
+license=('GPLv2')
+depends=('python')
+makedepends=(
+    'gcc'
```

Comment:
gcc, make, pkgconf, bash are not needed here as makedepends.

Reply:
> 



> If you read [spades_compile.sh](https://github.com/ablab/spades/blob/spades_3.15.5/assembler/spades_compile.sh), you could create this pkg by the common cmake build steps, something like this:
> 
>     1. `cmake -B build -S PKGSRC`, `PKGSRC` refers to the root dir contains the `CMakeLists.txt`, usually project source root.
> 
>     2. `make -C build`
> 
>     3. `make -C build DESTDIR=${pkgdir} install`
> 
> 
> The script `spades_compile.sh` seems to be used for user without root permission to compile and install this pkg to user's directory. It's not for creating a package for a Linux distribution.
> 
> See also https://wiki.archlinux.org/title/CMake_package_guidelines.
> 
> BTW, have you built it in a clean chroot as [this wiki](https://wiki.archlinux.org/title/DeveloperWiki:Building_in_a_clean_chroot) describes? It seems there is issue with `lto`, you may need to add `options=(!lto)` to the `PKGBUILD`. Or even better, report an issue to the upstream. @starsareintherose Am I right we are using the default `/etc/makepkg.conf`?

Yes, totally agree with you. CMake's method is more clear, so that there is no need to link so many files.